### DISCUSSION TITLE
[Feature] Sync shared album wit contents from iOS photos app

### DISCUSSION THREAD

**Discussion by traktuner** [immich-app/immich#3932] [Feature Request]
### The feature

Hello,

I have my albums organized in the native iOS photos app.
I would like to:
go to the Library page in the Immich app, open an "album on device" and share that specific album with a person.
Additionally, the contents of that shared album should be the same as in the native iOS photos app -> when I add a picture/video in the iOS photos app, the person I shared the album with in Immich should also see the added photos/videos (so the shared album in Immich should be synced with the iOS photos app album).
Since I have really good organized albums on iOS I would have to do doulbe the work and create the corresponding albums in Immich separately.

Thank you!

### Platform

- [ ] Server
- [ ] Web
- [X] Mobile

**Comment by bo0tzz** [comment]
Duplicate of #1678
